src/ddscxx/src/org/eclipse/cyclonedds/topic/AnyTopicDelegate.cpp

```diff
@@ -192,9 +192,9 @@ AnyTopicDelegate::discover_topics(
     ISOCPP_THROW_EXCEPTION(ISOCPP_UNSUPPORTED_ERROR, "Discovery of AnyTopics not implemented yet");
 }
 
-void AnyTopicDelegate::set_copy_out(org::eclipse::cyclonedds::topic::copyOutFunction copyOut)
+void AnyTopicDelegate::set_copy_out(org::eclipse::cyclonedds::topic::copyOutFunction _copyOut)
 {
-    this->copyOut = copyOut;
+    this->copyOut = _copyOut;
 }
```

Comment:
We can probably remove `set_copy_out`?

Reply:
Removed this function, since it is not used anywhere.